src/lib/utils/index.ts

```diff
@@ -261,3 +262,33 @@ export const assertGetChainAdapter = (
 
   return adapter
 }
+
+export const ALLOWED_PRICE_IMPACT_PERCENTAGE_LOW: string = '1' // 1%
+export const ALLOWED_PRICE_IMPACT_PERCENTAGE_MEDIUM: string = '3' // 3%
+export const ALLOWED_PRICE_IMPACT_PERCENTAGE_HIGH: string = '5' // 5%
+export const ALLOWED_PRICE_IMPACT_PERCENTAGE_EXPERT: string = '15' // 15%
+
+const IMPACT_TIERS = [
+  ALLOWED_PRICE_IMPACT_PERCENTAGE_EXPERT,
+  ALLOWED_PRICE_IMPACT_PERCENTAGE_HIGH,
+  ALLOWED_PRICE_IMPACT_PERCENTAGE_MEDIUM,
+  ALLOWED_PRICE_IMPACT_PERCENTAGE_LOW,
+]
+
+type WarningSeverity = 0 | 1 | 2 | 3 | 4
+
+export function warningSeverity(priceImpact: string | undefined): WarningSeverity {
```

Comment:
Is `priceImpact` a percentage? If so, can we name it as such?

Reply:
good call, i've aligned the naming to `priceImpactPercentage`